lib/macho/exceptions.rb

```diff
@@ -3,6 +3,14 @@ module MachO
   class MachOError < RuntimeError
   end
 
+  # A generic and non-fatal Mach-O error during file modification.
```

Comment:
I think we can drop `generic` from the description. It doesn't seem to add any value.